packages/component-library/package.json

```diff
@@ -2,11 +2,10 @@
   "name": "@ducky/plumage",
   "version": "3.3.0",
   "description": "Plumage is Ducky's design system",
-  "main": "dist/cjs/plmg.cjs.js",
+  "main": "dist/index.cjs.js",
```

Comment:
This as a warning / advised update that we've ignore for a while. I changed it and nothing broke and the warning went away so left as is

Reply:
Can you keep note of these in future PRs so we know what the warning was and why it changed?
Right now I'm not sure what was expected here, and it not breaking locally doesn't give me a tonne of confidence 😅 
For example, the launch config in component-library was expecting `"program": "${workspaceFolder}/dist/cjs/plmg.cjs.js"`, and it's not working now. But I don't know if it was broken before too because I had never used it (doesn't mean other people don't)

Reply:
Will do. Fair point. Maybe switch it back for now to get this one done?

Reply:
Get out of here! Go have vacation!

Reply:
Just so you know what's changed, I revered those 2 changes to the package.json, I wasn't able to see warnings when doing this so everything should be fine?
I also added `--emptyOutDir` to the vite build script in the vite project, not that we use it. It just got rid of a console warning when building.
Lastly, I changed the `@types/node` from version 20 down to 18. These `type` version files should match the version of `node` you're using